src/components/Examples.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+import { exampleQueries } from '@/modules/exampleQueries';
+import { queryProvivderKey } from '@/stores/query';
```

Comment:
Typo?

Reply:
Yep, sometimes the autocomplete works too well :)